### PR TITLE
Makes nanite brainwashing disk an uplink item, removes from hazard tech.

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
I have moved to the nanite brainwashing program from illegal tech to a science restricted syndicate item that is functionally just a nanite disk with a unique sprite and preloaded with brainwashing tech (3tc). Also while we're here I added this sprite to the brainwashing surgery disk because it was conspicuous enough already. We're back in the game.

## Why It's Good For The Game
I believe the intention of the brainwashing program was to be used primarily for antagonistic purposes. However, I have only ever seen this utilized by security or whoever to make antagonists turn themselves in, kill themselves, or act like wage slaves. I doubt this is a desired or intended result of this program and we already have a better item for this in the form of the mind-control chair. This allows the program to be used for its intended purpose without these undesired effects. Now, do the fact that it may be more costly to acquire this program as a syndie I have contemplated buffing this, perhaps with a reduced TC cost or a longer time in which your target is mind-controlled. Also, Syndie sprites are cool.

## Changelog
:cl: itseasytosee
del: Nanotrasen scrubbed their nanite servers presently to remove a syndicate brainwashing program that snuck past the spiders
add: A syndicate brainwashing mind control program can be purchased for traitor scientists 
/:cl:

### CODE REVIEW

**Comment by CarpStation** [comment]
test

**Comment by CRITAWAKETS** [comment]
"Hazard Nanite Programs" is alien technology, not illegal technology. Illegal nanites are "Military Nanite Programs".

File: code/modules/research/nanites/program_disks.dm [hunk 1]
```diff
@@ -12,6 +12,12 @@
 	if(program_type)
 		program = new program_type
 
+/obj/item/disk/nanite_program/mind_control
+	name = "suspicious-looking disk"
+	desc = "A sinisterly colored nanite program disk. Comes preloaded with a brainwashing program."
+	icon_state = "datadisk_syndie"
+	program_type = /datum/nanite_program/comm/mind_control
```
Comment (ATH1909) [inline code/modules/research/nanites/program_disks.dm:19]: it's really gonna suck to accidentally write over this program

Comment (itseasytosee) [reply code/modules/research/nanites/program_disks.dm:19]: I put a warning in the uplink screen

**Comment by Angustmeta** [comment]
9tc is way too expensive imo. It takes effort, knowledge and timegates (nanite research + illegal tech) to forcefully infect someone with nanites (nanite sting). 3 tc feels more prudent. 

Brainwash surgery disk and hypnoflash both take less thinking to get them to work, and are permanent.

**Comment by Fikou** [comment]
this is alien tech

**Comment by Dennok** [comment]
Yes, this is alien tech. 
You can add it as traitor item, but please, dont remove it from extremaly rare, and hard to get, last tech nodes of researtch.

**Comment by Pugie** [comment]
Either reduce price or increasw hypnotic flash price otherwise this is a good change.

**Comment by EgorDinamit** [comment]
First of all - make sure you can't delete this program from the disk.
Then I'd recommend lowering the price to 2-4 TC.

Right now using chain-detonation seems like a better option only for 1 TC and bit of research.

**Comment by itseasytosee** [comment]
I thought I would have to reduce the TC but I'm glad to have a few more opinions on it
@EgorDinamit  I don't see why I should make the program undeletable that makes no sense consistency wise. 
@Dennok I don't think this is a proper reward for acquiring alien tech as I said there are very few ways to utilize this as a crewmate in a way that doesn't go against the spirit of the game. Advanced operating tools are an example of a good reward as they generally help the crew and can also be utilized by antagonists. As for lore consistency, I think that the syndicate has quite a few brainwashing items and it would not be out of place to move it completely. I would be totally for adding a new nanite type to alien tech that might be utilized in a better way like not needing a heart (the alien operating table dose this for you so it's not a stretch) or perhaps telepathy (although its a bit overdone.)

**Review by Twaticus — APPROVED** [review]

**Comment by EgorDinamit** [comment]
I just imagine someone accidentally choosing another program while on brainwashing disk, wasting 3 TC.
Good thing it's 3 now, instead of 9.

**Comment by XDTM** [comment]
>I believe the intention of the brainwashing program was to be used primarily for antagonistic purposes.

Speaking as the one who made it, the intention of this and the other programs is to have a very powerful tool in the hands of whoever controls it, and that does include using it _against_ antagonists (which should really know better than getting nanites if brainwashing is a possibility).
The balancing point being, of course, that once that tool is in the field, an antagonist could turn it against the station for disastrous effect, especially if nanites are widespread enough to catch random antags hiding within the crew.

On top of this, i really really hate the idea of restricting such a program to one single antagonist type, since it prevents it from interacting with all other types of antags. A creative cult might want to brainwash people into joining them in an orderly line with nanites, but they don't have an uplink and the gimmick is shot down.

With all that said, i'd be ok with adding the program to the uplink as long as the researchable option is not removed. I've read a lot of stories about chain detonation, but i'd like seeing more nanite mind control gimmicks going around, since they have more space for creativity.

**Pull request closed without merging**